### DISCUSSION TITLE
Increase poll intervall

### DISCUSSION THREAD

**Discussion by ballonchef** [StephanJoubert/home_assistant_solarman#646] [Q&A] [answered]
Dear all,

I successfully connected HO to mit ZCS Azzurro HYD 15000 using the sofar_g3hyd.yaml. This HACS integration is so great and I am very thankful to Stephan Joubert for this.

Home assistants refreshes the data from the inverter every 30 sec. which is great for "normal" use.

I also use this information for charging my electric car. In this case an update every second or every 5 seconds would help a lot to better adjust the charging rate. Is there any possibility to (temporarily) adjust the polling frequency?

**Comment by PV-Joe** [comment, marked as answer]
Install this Version and have 5 seconds update interval. You can choose how you want it for each sensor as well.

https://github.com/davidrapan/ha-solarman

**Reply by ballonchef** [reply]
Cool, great and thanks. Does it have any negative effects on HA or the inverter, when the interval is more frequent?

**Reply by PV-Joe** [reply]
> Cool, great and thanks. Does it have any negative effects on HA or the inverter, when the interval is more frequent?

No, not at all. 👍
